examples/fabric-admin/device_manager/DeviceManager.cpp

```diff
@@ -284,7 +342,7 @@ void DeviceManager::HandleAttributeData(const app::ConcreteDataAttributePath & p
         if (value.Has(CommissionerControl::SupportedDeviceCategoryBitmap::kFabricSynchronization))
         {
             ChipLogProgress(NotSpecified, "Remote Fabric-Bridge supports Fabric Synchronization, start reverse commissioning.");
-            StartReverseCommissioning();
+            RequestCommissioningApproval();
```

Comment:
Outside of this PR, but why are we doing this on successfully reading the attribute and not as an independent command in fabric admin?

Reply:
This first check of the device categories specified in SupportedDeviceCategoryBitmap has kFabricSynchronization, 

if Yes, it call RequestCommissioningApproval() which do send an independent command RequestCommissioningApproval

Reply:
This is automated based on reading a value. If fabric-admin does `commissionercontrol read supported-device-categories` we then trigger this entire process which seems like it is not something we want to do

Reply:
Why? we just auto-consent in the example app to exercise the flow, since the example app is CLI based, it is quite challenging to simulate all UI interactions with the end user.

Reply:
It give us more control. I don't see why you wouldn't just manually do `commissionercontrol request-commissioning-approval ...`

Like in CI, or in the TH we can use pipes to have more control in CI/TH as to when we do things instead of just having a bunch of things done for us just because we read an attribute at one point.

Also what happens if you read this attribute a second time you are starting this all over again

Reply:
Again this can be done in a follow up, but we should have a github issue for tracking

Reply:
https://github.com/project-chip/connectedhomeip/issues/35041

I believe simulating user interaction should be left for the final product integration. Including these steps in the example app will complicate the automation process. Since we don't intend to commercialize the example app, let's not prioritize this task and instead focus on completing the remaining features.